crypto/x509/x509_vfy.c

```diff
@@ -3316,20 +3237,26 @@ static int build_chain(X509_STORE_CTX *ctx)
     default:
         num = sk_X509_num(ctx->chain);
         CB_FAIL_IF(num > depth,
-                   ctx, NULL, num-1, X509_V_ERR_CERT_CHAIN_TOO_LONG);
+                   ctx, NULL, num - 1, X509_V_ERR_CERT_CHAIN_TOO_LONG);
         CB_FAIL_IF(DANETLS_ENABLED(dane)
                        && (!DANETLS_HAS_PKIX(dane) || dane->pdpth >= 0),
-                   ctx, NULL, num-1, X509_V_ERR_DANE_NO_MATCH);
+                   ctx, NULL, num - 1, X509_V_ERR_DANE_NO_MATCH);
         if (self_signed)
-            return verify_cb_cert(ctx, NULL, num-1,
-                                  sk_X509_num(ctx->chain) == 1
-                                  ? X509_V_ERR_DEPTH_ZERO_SELF_SIGNED_CERT
-                                  : X509_V_ERR_SELF_SIGNED_CERT_IN_CHAIN);
-        return verify_cb_cert(ctx, NULL, num-1,
-                              ctx->num_untrusted < num
-                              ? X509_V_ERR_UNABLE_TO_GET_ISSUER_CERT
-                              : X509_V_ERR_UNABLE_TO_GET_ISSUER_CERT_LOCALLY);
-    }
+            return verify_cb_cert(ctx, NULL, num - 1,
+                                  sk_X509_num(ctx->chain) == 1 ?
+                                  X509_V_ERR_DEPTH_ZERO_SELF_SIGNED_CERT :
+                                  X509_V_ERR_SELF_SIGNED_CERT_IN_CHAIN);
```

Comment:
I prefer keeping the ? and : at the beginning of the next line.

Reply:
Done